README.template.md

```diff
@@ -0,0 +1,139 @@
+# [TODO:PROJECT_NAME]
+
+[TODO:PROJECT_DESCRIPTION]
+
+- [Setup](#setup)
+  - [Prerequisites / Dependencies](#prerequisites--dependencies)
+  - [Installation](#installation)
+  - [Post-install](#post-install)
+- [Usage](#usage)
+  - [Running the app](#running-the-app)
+  - [Building the app](#building-the-app)
+- [Testing](#testing)
+  - [Unit testing](#unit-testing)
+  - [End to end tests](#end-to-end-tests)
+  - [Code quality](#code-quality)
+- [Environments](#environments)
+- [Environment variables](#environment-variables)
+  - [List of environment variables](#list-of-environment-variables)
+- [Deployment](#deployment)
+- [Releases](#releases)
+- [Translations](#translations)
+- [Licencing](#licencing)
+
+## Setup
+
+### Prerequisites / Dependencies
+
+You will need to have NodeJS and NPM installed to run this application. It's recommended to use a NodeJS version manager to manage your versions. Required NodeJS versions will typically match LTS (long-term support) versions and you can check which exact version to install by checking `.node-version` file in the repository root.
+
+### Installation
+
+1. clone this repository: `git clone [TODO:GIT_LINK]`
+2. install all the dependencies: `npm ci`
+
+### Post-install
+
+1. set up the [environment variables](#environment-variables)
+
+## Usage
+
+### Running the app
+
+To run the app, you can simply run `npm start`. This starts the application in the environment dependent on your environment setup.
+
+### Building the app
+
+To build the app you can use some of the following commands:
+
+```
+npm run build
+npm run build:dev
+npm run build:prod    # builds the app for production
+
+```
+
+## Testing
+
+### Unit testing
+
+To start the unit tests you can use one of the following commands:
+
+```
+npm run test
+npm run test:watch              # runs all tests in watch mode and without code coverage report
+npm run test:watch:coverage     # runs all tests in watch mode with code coverage report
+
+```
+
+### End to end tests
+
+[TODO:DESCRIBE_E2E]
+
+### Code quality
+
+The following tools are used in the project to ensure code quality:
+
+Prettier: for consistent code formatting.
+ESLint: for identifying and fixing code quality issues.
+Stylelint: for identifying and fixing CSS code quality issues.
```

Comment:
We don't use Stylelint on React, since we're not using SCSS/CSS directly

Reply:
Removed